### ISSUE TITLE
Streaming presenter names and profiles

### ISSUE THREAD

**Issue by amyjko** [clowdr-app/clowdr#342]
**Is your feature request related to a problem? Please describe.**
One of the excellent features in video chat rooms is the ability to see all speakers names, and click on their names to see their profile. It provides excellent social context to who is speaking. However, none of this information is available during a streamed presentation. For example, we were in a session with one session chair and six presenters; no one in the audience knew who was who, and the session chair didn't ask everyone to introduce themselves, so most of the newcomers in the audience watched confused about who these people were and what expertise they had.

**Describe the solution you'd like**
Include the names of presenters in the stream in their videos, and if possible, a link to their profiles, mirroring the functionality in rooms.

**Comment by rossng** [comment]
I completely agree - this is something we'd really like to enable. We would have added it if not for a technical limitation - Clowdr's WebRTC provider doesn't support this natively.

We have very recently added profile pictures of the presenters/chairs to the room page, which hopefully provides a small mitigation for not knowing who's on stage.

**Comment by rossng** [comment]
There is a possible workaround that would allow us to add basic nametags to the stream. We can use an HTML canvas to composite the camera video with some text, and then capture it back into another video stream. This can then be passed into the WebRTC room as a substitute for the 'actual' camera video.

However, the canvas-based compositing is costly in CPU usage. This might degrade the quality of some presenters' video - we have already seen machines that struggle in the backstage even without additional processing.

A minor additional consideration is that the text label would also be seen by other backstage participants - and would likely clash with the existing interactive name label.

**Comment by amyjko** [comment]
Another odd but potentially compelling design idea would be to highlight the name of the active speaker in the page metadata, rather than the video. It would be disjoint, which isn't ideal, but it might have the added benefit of helping bring attention to the metadata (which many attendees haven't noticed because most of it is below the fold).
